k8s/namespaces/sscs/sscs-cor-frontend/aat.yaml

```diff
@@ -4,26 +4,17 @@ metadata:
   name: sscs-cor-frontend
   namespace: sscs
 spec:
-  releaseName: sscs-cor-frontend
   values:
     nodejs:
-      replicas: 2
-      useInterpodAntiAffinity: true
-      image: hmctspublic.azurecr.io/sscs-cor/frontend:prod-23f0ef0-20220513104022 #{"$imagepolicy": "flux-system:sscs-cor-frontend"}
       ingressHost: sscs-cor.aat.platform.hmcts.net
       environment:
         NODE_ENV: preprod
-        MYA_FEATURE_FLAG: "true"
         ADDITIONAL_EVIDENCE_FEATURE_FLAG: "true"
         EVIDENCE_UPLOAD_QUESTION_PAGE_OVERRIDE_ALLOWED: "true"
-        POST_BULK_SCAN: "true"
-        FT_WELSH: "true"
         HEARING_OUTCOME_TAB_ENABLED: "true"
         MEDIA_FILES_ALLOWED_ENABLED: "true"
         REQUEST_TAB_ENABLED: "true"
         ALLOW_COOKIE_BANNER_ENABLED: "true"
         DUMMY_PROPERTY: "true"
     global:
       environment: aat
```

Comment:
remove

Reply:
Removed

Reply:
best to not comment till you've pushed unless it's a question :)